packages/test/test-utils/src/testContainerRuntimeFactory.ts

```diff
@@ -61,22 +62,29 @@ export const createTestContainerRuntimeFactory = (
 			context: IContainerContext,
 			existing: boolean,
 		): Promise<IRuntime & IContainerRuntime> {
-			const runtime: ContainerRuntime = await containerRuntimeCtor.load(
+			const runtime: ContainerRuntime = await containerRuntimeCtor.loadRuntime({
```

Comment:
I think we should have fixed this some time ago. I need to make these changes eventually anyway.

Reply:
Fixing this causes some attribution tests to fail, reverted this change